ArduCopter/events.cpp

```diff
@@ -14,10 +14,23 @@ void Copter::failsafe_radio_on_event()
     if (should_disarm_on_failsafe()) {
         init_disarm_motors();
     } else {
-        if (control_mode == AUTO && g.failsafe_throttle == FS_THR_ENABLED_CONTINUE_MISSION) {
-            // continue mission
+        if (control_mode == AUTO && g2.fs_thr_options & ParametersG2::FS_THR_OPT_MISSION_CONTINUE) {
```

Comment:
Use parentesis for readability around (g2.fs_thr_options & ParametersG2::FS_THR_OPT_MISSION_CONTINUE)